src/electionguard/logs.py

```diff
@@ -17,12 +17,22 @@ class ElectionGuardLog(Singleton):
 
     __logger: logging.Logger
 
-    def __init__(self) -> None:
+    def __init__(
+        self,
+        handlers: List[logging.Handler] = None,
```

Comment:
I may already be rusty at my python, but does this need to be wrapped in `Optional` to accept a `None` value?